.github/workflows/automatic-api-update.yaml

```diff
@@ -32,11 +32,11 @@ jobs:
       - name: "Install ruby"
         uses: "ruby/setup-ruby@v1"
         with:
-          ruby-version: "2.7.3"
+          ruby-version: "3.2.2"
```

Comment:
I think we were previously targeting an older but still commonly used version of Ruby and not the latest. @samkim do you remember how we got this version?

Reply:
I can rollback this change, I though it was worth starting to test 3.X

Reply:
@jzelinskie reverted: https://github.com/authzed/authzed-rb/pull/76/commits/1adf79dba55e472dcec8fb12ec046de663c39914